internal/osutils/osutils_windows.go

```diff
@@ -27,6 +28,22 @@ func CmdExitCode(cmd *exec.Cmd) (code int) {
 	return cmd.ProcessState.Sys().(Status).ExitStatus()
 }
 
+// BashifyPath takes a windows %PATH% list and turns it into a bash style PATH list.
+// e.g. C:\foo;C:\bar becomes /c/foo:/c/bar
+func BashifyPathEnv(pathList string) string {
+	dirs := strings.Split(pathList, ";")
+	for i, dir := range dirs {
+		// Calling out to bash like in BashifyPath is too slow. Just do a simple string transformation.
+		// The resulting paths will be valid in bash, they just may not be fully resolved/simplified.
+		// This is okay because bash really just needs to find executables to run.
+		vol := strings.ToLower(filepath.VolumeName(dir)) // "C:\foo bar\baz" -> "c:"
+		dir = dir[len(vol):]                             // "C:\foo bar\baz" -> "\foo bar\baz"
+		vol = strings.Replace(vol, ":", "", 1)           // "c:" -> "c"
+		dirs[i] = "/" + vol + filepath.ToSlash(dir)      // "C:\foo bar\baz" -> "/c/foo bar/baz"
+	}
+	return strings.Join(dirs, ":") // bash uses ':' while Windows uses ';'
+}
```

Comment:
This is assuming drive letters are mounted as `/c/`. I've definitely used bash version on Windows where this was not true (I think cygwin is an example of this, they might use `/mnt/c/` or `/drive/c` or something along those lines.

String replacement alone simply cannot sufficiently address this use-case. Please consider the solution I suggested here: https://github.com/ActiveState/cli/pull/2189#discussion_r1012042815

Reply:
I am using a variant of the suggested solution in that comment. See my last comment there.